civitai-downloader-v2/src/cli/main.py

```
@@ -169,8 +169,552 @@ async def run_search():
+                    if category or tags or base_model:
+                        with open(intermediate_path, 'w', encoding='utf-8') as raw_f:
+                            raw_results_dict = [res.dict() if hasattr(res, 'dict') else res for res in results]
+                            json.dump(raw_results_dict, raw_f, indent=2, ensure_ascii=False)
+                    
+                    # Save filtered data to main file with cleaned descriptions
+                    filtered_results_dict = [res.dict() if hasattr(res, 'dict') else res for res in filtered_results]
+                    # Clean HTML from description fields
+                    cleaned_results = []
+                    for result in filtered_results_dict:
+                        if isinstance(result, dict):
+                            cleaned_result = HTMLCleaner.clean_model_description(result.copy())
```

Ruff (0.12.2)

Undefined name `HTMLCleaner`

(F821)